hadoop-ozone/dist/src/main/smoketest/ozone-lib/freon.robot

```diff
@@ -67,3 +67,9 @@ Freon OMBR
     [arguments]    ${prefix}=ombg    ${n}=1    ${threads}=1    ${args}=${EMPTY}
     ${result} =        Execute          ozone freon ombr ${OM_HA_PARAM} -t ${threads} -n${n} -p ${prefix} ${args}
                        Should contain   ${result}   Successful executions: ${n}
+
+
+Freon S3BG
+    [arguments]    ${prefix}=s3bg    ${n}=100    ${threads}=5000    ${args}=${EMPTY}
+    ${result} =        Execute          ozone freon s3bg ${OM_HA_PARAM} -t ${threads} -n ${n} -p ${prefix} ${args}
+                       Should contain   ${result}   Successful executions: ${n}
```

Comment:
@Zeddling Sorry, I just noticed this.  I think this almost duplicates the one in `s3/freon.robot` and is unused.  Let's keep only the other one.

```suggestion
```

Reply:
Oh sorry. I missed that. Thank you